requirements.txt

```diff
@@ -0,0 +1,8 @@
+certifi==2021.10.8
+chardet==4.0.0
+charset-normalizer==2.0.12
+idna==3.3
+Pillow==9.0.1
+requests==2.27.1
+tk==0.1.0
+urllib3==1.26.8
```

Comment:
is this file needed and by what kind of dependencies manager ?

Reply:
> is this file needed and by what kind of dependencies manager ?

That is the standard way to store python dependencies, its used by pip, those are all the packages your project needs to be installed in order to run on any system, the versions are the current versions as of when I wrote that file.

https://www.idkrtm.com/what-is-the-python-requirements-txt/